pkg/pillar/cmd/volumemgr/initialvolumestatus.go

```diff
@@ -144,20 +138,18 @@ func populateInitialVolumeStatus(ctx *volumemgrContext, dirName string) {
 		}
 
 		status := types.VolumeStatus{
-			BlobSha256:   sha256,
-			AppInstID:    appUUID,
-			VolumeID:     nilUUID, // XXX known for other origins?
-			PurgeCounter: purgeCounter,
-			DisplayName:  "Found in /persist/img",
-			FileLocation: filelocation,
-			State:        types.CREATED_VOLUME,
-			// XXX Is this the correct size? vs. qcow2 size?
-			TargetSizeBytes: uint64(size),
```

Comment:
TargetSizeBytes should track the current size (which we should extract from qemu-img for qcow2). Why leave it at zero here?

Reply:
While preparing initial status, we are not handling/filling sizes. At the time of volume creation, we are taking the sizes from the config.
Note: we are not taking the output of the qemu for filling TargetSizeBytes in master.

Reply:
But volumemgr needs to be able to track the disk usage, and if we keep it at zero in the initial VolumeStatus then how will volumemgr track the disk usage?

Reply:
Initial status gets overwritten with the actual config passed from the zedmanager in volumemgr at the time volume creation.

Reply:
OK